admin/site-admin/projects.md

```diff
@@ -0,0 +1,9 @@
+# Manage projects
+
+Access **Projects** from the Stoplight dashboard by selecting:
+
+**admin** > **Site Admin** > **Projects**
+
+You can view a list of projects and search for an individual project.
+
+<!-- QUERY: at present, the UI says "Projects are synced from external services". Will there be no way to create a project from scratch in Stoplight Dojo? -->
```

Comment:
Yes, no way to create a project in Stoplight.  We rely upon a backing VCS repo for all projects.